src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/repository/EventAuditRepository.kt

```diff
@@ -19,7 +19,7 @@ interface EventAuditRepository : JpaRepository<EventAudit, Long> {
       "ORDER BY id DESC LIMIT 1 ",
     nativeQuery = true,
   )
-  fun findLastBookedVisitEventByBookingReference(bookingReference: String): EventAudit
+  fun findLastBookedVisitEventByBookingReference(bookingReference: String): EventAudit?
```

Comment:
set this to NULL as this might return a NULL of the visit was migrated.

Reply:
Should never be null, migrated visits have Event audits!